### ISSUE TITLE
Module setup for AOT seems to do nothing

### ISSUE THREAD

**Issue by patryk-wlaz** [Wykks/ngx-mapbox-gl#241]
Method documented as appriopriate library usage for AOT builds doesn't work, which is reproduced here https://github.com/patryk-wlaz/ngx-mapbox-gl-error
Specifically: see lines commented out in this file https://github.com/patryk-wlaz/ngx-mapbox-gl-error/blob/master/src/app/app.module.ts

**Comment by tiukh** [comment]
@dmytro-gokun hi, do you have any updates on this?

after migration from angular 8 to 9 and ngx-mapbox from 3.3.1 to 4.8.1, got an error

```
Warning: Invalid constructor parameter decorator in ../node_modules/ngx-mapbox-gl/fesm2015/ngx-mapbox-gl.js:
 () => [
    { type: NgZone },
    { type: String, decorators: [{ type: Optional }, { type: Inject, args: [MAPBOX_API_KEY,] }] },
    { type: MglResizeEventEmitter, decorators: [{ type: Optional }] }
]
Error: Error on worker #8: Error: getInternalNameOfClass() called on a non-ES5 class: expected MapService to have an inner class declaration
```

**Comment by dmytro-gokun** [comment]
@tiukh No, unfortunately, I'm very tight on time now. If you are willing to step up and help with finding the root caise and - you are very welcome. As well as anyone else.

**Comment by dmytro-gokun** [comment]
@tiukh Does this issue exist with https://github.com/Wykks/ngx-mapbox-gl/releases/tag/v5.0.0?
If yes, please provide a reproduction.
